server/lib/resolvers/builders/index.js

```diff
@@ -2,5 +2,6 @@
 module.exports = {
   // Matches the DataSource type ENUM in the models/dataSource.js
   InMemory: require('./nedb'),
-  Postgres: require('./postgres')
+  // Postgres: require('./postgres'),
```

Comment:
+1 to remove Postgres SQL resolvers as they are no providing standard security etc.